Symfony/app/Resources/views/base.html.twig

```diff
@@ -42,7 +42,15 @@
                         <a href="#contact">Contact</a>
                     </li>
                     <li>
-                        <a href="#login">Login</a>
+                    {% if is_granted('ROLE_USER') %}
+                        <a href="./logout">Logout</a>
+                    {% endif %}
+                        {% if not is_granted('ROLE_USER') %}
```

Comment:
The rule about `else` stands mostly for php/js code and not twigs. The idea is to make things simpler and cleaner, if removing an `else` causes far more complicated code then it's definitely not worth it.